writeengine/bulk/we_brmreporter.cpp

```diff
@@ -413,7 +414,7 @@ int BRMReporter::openRptFile( )
         return rc;
     }
 
-    fRptFile << "#CP:   startLBID max min seqnum type colwidth newExtent"    << std::endl;
+    fRptFile << "#CP:   startLBID seqnum type colwidth max min newExtent"    << std::endl;
```

Comment:
Max and min values should be after type and colWidth - these two determine types used.